runtime/runtime/src/state_viewer/mod.rs

```diff
@@ -170,14 +171,18 @@ impl TrieViewer {
         let mut values = vec![];
         let query = trie_key_parsers::get_raw_prefix_for_contract_data(account_id, prefix);
         let acc_sep_len = query.len() - prefix.len();
-        let mut iter = state_update.trie().disk_iter()?;
-        iter.remember_visited_nodes(include_proof);
```

Comment:
We got rid of `remember_visited_nodes` in favor of using recorded storage, which makes a lot more sense.

Reply:
This doesn't seem to be working as expected, I'll take a detailed look.